src/styles/blocks/stars_modifier.css

```diff
@@ -0,0 +1,19 @@
+.stars--1 .stars_star:nth-child(-n + 1) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--2 .stars_star:nth-child(-n + 2) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--3 .stars_star:nth-child(-n + 3) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--4 .stars_star:nth-child(-n + 4) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--5 .stars_star:nth-child(-n + 5) {
+  background-image: url(../../images/star-active.svg);
```

Comment:
You're repeating the `background-image` property with the same value for different selectors. Consider using a common class for the active stars to reduce code duplication. For example, you could have a single rule like `.stars_star--active { background-image: url(../../images/star-active.svg); }` and apply this class to the active stars in your HTML. This will make your CSS more maintainable and reduce the file size.